<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>726</width>
    <height>1071</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="connection_box">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>701</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
background-color: rgba(0, 0, 139, 0.1);
    border-radius: 5px; /* Arrotonda gli angoli del pulsante */</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="disconnectButton">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>140</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    border-radius: 10px; /* Arrotonda gli angoli del pulsante */
    background-color: #FFFFFF; /* Imposta il colore di sfondo del pulsante */
}
</string>
     </property>
     <property name="text">
      <string>Disconnect</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="label_ip">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>113</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;
 background-color: transparent; /* Imposta il colore di sfondo del pulsante */
</string>
     </property>
     <property name="text">
      <string>SV Board IP Address:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="label_boardstatus">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>113</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;
 background-color: transparent; /* Imposta il colore di sfondo del pulsante */
</string>
     </property>
     <property name="text">
      <string>Board Status</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="IPentry">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="type_otx">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>10</y>
       <width>82</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>OTX</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="port_entry">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>50</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="label_radc">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>10</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;
 background-color: transparent; /* Imposta il colore di sfondo del pulsante */
</string>
     </property>
     <property name="text">
      <string>RADC</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="status">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>130</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QRadioButton" name="type_orx">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>10</y>
       <width>82</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>ORX</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="mac_entry">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>90</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="connectButton">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>140</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    border-radius: 10px; /* Arrotonda gli angoli del pulsante */
    background-color: #FFFFFF; /* Imposta il colore di sfondo del pulsante */
}
</string>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="label_port">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>113</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;
 background-color: transparent; /* Imposta il colore di sfondo del pulsante */

</string>
     </property>
     <property name="text">
      <string>Trget Port</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="label_mac">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>113</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;
 background-color: transparent; /* Imposta il colore di sfondo del pulsante */
</string>
     </property>
     <property name="text">
      <string>MAC Address</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="manua_commandbox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>701</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
background-color: rgba(80, 10, 139, 0.3);
    border-radius: 5px; /* Arrotonda gli angoli del pulsante */</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="label_manualcommand">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>113</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;
 background-color: transparent; /* Imposta il colore di sfondo del pulsante */
</string>
     </property>
     <property name="text">
      <string>Manual Command</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>221</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_13">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>0</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;
 background-color: transparent; /* Imposta il colore di sfondo del pulsante */
</string>
     </property>
     <property name="text">
      <string>Channel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="sendcommand">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>10</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    border-radius: 10px; /* Arrotonda gli angoli del pulsante */
    background-color: #FFFFFF; /* Imposta il colore di sfondo del pulsante */
}
</string>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="channelselection">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>10</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <zorder>label_manualcommand</zorder>
    <zorder>lineEdit_13</zorder>
    <zorder>sendcommand</zorder>
    <zorder>channelselection</zorder>
    <zorder>comboBox</zorder>
   </widget>
   <widget class="QFrame" name="LOG_BOX">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>701</width>
      <height>291</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
background-color: rgba(0, 0, 139, 0.1);
    border-radius: 5px; /* Arrotonda gli angoli del pulsante */</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>681</width>
       <height>221</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
background-color: white;
    border-radius: 5px; /* Arrotonda gli angoli del pulsante */</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="CH_TEXT_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;
 background-color: transparent; /* Imposta il colore di sfondo del pulsante */
</string>
     </property>
     <property name="text">
      <string>LOG BOX</string>
     </property>
    </widget>
    <widget class="QPushButton" name="clear_log">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    border-radius: 10px; /* Arrotonda gli angoli del pulsante */
    background-color: #FFFFFF; /* Imposta il colore di sfondo del pulsante */
}
</string>
     </property>
     <property name="text">
      <string>Clear log</string>
     </property>
    </widget>
    <widget class="QPushButton" name="save_log">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>260</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    border-radius: 10px; /* Arrotonda gli angoli del pulsante */
    background-color: #FFFFFF; /* Imposta il colore di sfondo del pulsante */
}
</string>
     </property>
     <property name="text">
      <string>Save log</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="NewIP_box">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>701</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
background-color: rgba(0, 0, 139, 0.1);
    border-radius: 5px; /* Arrotonda gli angoli del pulsante */</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="set_newip_button">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>21</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    border-radius: 10px; /* Arrotonda gli angoli del pulsante */
    background-color: #FFFFFF; /* Imposta il colore di sfondo del pulsante */
}
</string>
     </property>
     <property name="text">
      <string>Set &amp; Connect</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="ip">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="ip_2">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="ip_3">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>20</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="ip_4">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>20</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="label_newip">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>113</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;
 background-color: transparent; /* Imposta il colore di sfondo del pulsante */
</string>
     </property>
     <property name="text">
      <string>Set Ip address</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>726</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
